pom.xml

```diff
@@ -123,7 +123,7 @@
     <flume.version>1.6.0</flume.version>
     <zookeeper.version>3.4.6</zookeeper.version>
     <curator.version>2.6.0</curator.version>
-    <hive.group>org.spark-project.hive</hive.group>
+    <hive.group>com.github.hyukjinkwon</hive.group>
```

Comment:
I am going to revert this change and the changes in ` dev/run-tests.py` back to `org.spark-project.hive` once the tests pass. I redistributed `org.spark-project.hive` having one liner fix for Hadoop 3 support in order to use it in this PR.